core/src/components/picker-legacy/picker.tsx

```diff
@@ -30,7 +31,7 @@ import type { PickerButton, PickerColumn } from './picker-interface';
  * @virtualProp {"ios" | "md"} mode - The mode determines which platform styles to use.
  */
 @Component({
```

Comment:
Should we consider renaming the class name for `Picker` and `PickerColumnCmp` so that after migrating the internal implementation we don't have competing names for the modern vs. deprecated approaches?

Reply:
Wouldn't that only be a concern if you're developing Ionic and importing both the deprecated and modern approaches in the same folder? IIRC the class names aren't exposed to developers.

Reply:
Informational question: I assume we _do_ want to rename the `PickerInternal` class to just `Picker` even though it's not exposed to developers? From this, it sounds like that shouldn't cause any issues.

Reply:
> Informational question: I assume we do want to rename the PickerInternal class to just Picker even though it's not exposed to developers? From this, it sounds like that shouldn't cause any issues.

Yeah I think renaming `PickerInternal` to `Picker` makes sense. It shouldn't have any impact on developers, but I think from a maintenance point of view it could be confusing for a team member to see "Internal" on a public component.